core/src/com/unciv/logic/map/tile/Tile.kt

```diff
@@ -495,7 +496,7 @@ open class Tile : IsPartOfGameInfoSerialization {
     /** Implements [UniqueParameterType.TerrainFilter][com.unciv.models.ruleset.unique.UniqueParameterType.TerrainFilter] */
     fun matchesSingleTerrainFilter(filter: String, observingCiv: Civilization? = null): Boolean {
         return when (filter) {
-            "All" -> true
+            "All", "Terrain" -> true
```

Comment:
Under what situations would you use "Terrain"?

Reply:
https://github.com/yairm210/Unciv/issues/10868#issuecomment-1877771043